.drone.yml

```diff
@@ -29,7 +29,7 @@ pipeline:
     image: antora/antora:1.0.1
     pull: true
     commands:
-      - antora generate --cache-dir cache/ --ui-bundle-url ui-bundle.zip --redirect-facility netlify site.prod.yml
+      - antora generate --cache-dir cache/ --ui-bundle-url ui-bundle.zip --redirect-facility netlify site.yml
```

Comment:
ui-bundle should be dropped as well, that should be part of the playbook.

Reply:
What are we setting this to? We didn't finalise this.

Reply:
Look at my docs-ui pull request, the bundle gets uploaded to our Minio on every push to master. If you tag a git version it automatically builds the bundle and attaches it to a github release.

The only thing you got to do manually is writing the github release title and comment.

Reply:
Still need to drop the bundle flag, and maybe even delete the bundle from this repo.

Reply:
I'll do that in a follow-up PR.